app/api/migrations/migrations/36-populate-mimetype-on-attachments/index.js

```diff
@@ -0,0 +1,31 @@
+import request from 'shared/JSONRequest';
+import { attachmentsPath } from 'api/files/filesystem';
+import mime from 'mime-types';
+
+export default {
+  delta: 36,
+
+  name: 'populate-mimetype-to-attachment',
+
+  description: 'Populates mimetype of an attachment from a url',
+
+  async up(db) {
+    process.stdout.write(`${this.name}...\r\n`);
+    const cursor = await db.collection('files').find({});
+
+    // eslint-disable-next-line no-await-in-loop
+    while (await cursor.hasNext()) {
+      // eslint-disable-next-line no-await-in-loop
+      const file = await cursor.next();
+      if (file.url && !file.mimetype) {
+        // eslint-disable-next-line no-await-in-loop
+        const response = await request.head(file.url);
+        const mimetype = response.headers.get('content-type') || undefined;
+        db.collection('files').updateOne({ _id: file._id }, { $set: { mimetype } });
+      } else if (file.filename && file.type === 'attachment' && !file.mimetype) {
+        const mimetype = mime.lookup(attachmentsPath(file.filename));
```

Comment:
As far as I know `lookup` will return `false` if the type cannot be determined. We definitivelly need a default here like `.lookup(...) || undefined` (the default is to be defined, see previous comment)
cc. @konzz

Reply:
Effectively, `false` it's been saved to the database in those cases.

Reply:
@grafitto, ok, lets use undefined for the default, but still we need to implement it as default value as noted in the previous comment :) thanks!

Reply:
Done